tools/setup_helpers/cmake.py

```diff
@@ -165,6 +165,10 @@ def generate(self, version, cmake_python_library, build_python, build_test, my_e
                 USE_NINJA and not os.path.exists(ninja_build_file)):
             # Everything's in place. Do not rerun.
             return
+        ninja_deps_file = os.path.join(self.build_dir, '.ninja_deps')
+        if IS_WINDOWS and USE_NINJA and os.path.exists(ninja_deps_file):
+            # Cannot rerun ninja on Windows
```

Comment:
Could you make the comment clearer that this is a workaround due to a ninja bug?

Reply:
@xuhdev Done!